### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Consistent Identification: Same icon is used for the identification of "Select Folder" button and "More" menu button.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#129]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS3.2.4

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Tab till "Select Folder" button and Press Enter.
7. Verify the consistent identification of UI components on the page.

**Actual Result:**
Same icon is used for the identification of "Select Folder" button and "More" menu button.

**Expected Result:**
Only the UI components which is having the same functionality should visually appear the same. Here "More" menu button and "Select Folder" button are two different controls with different functionality, but they visually appear the same.

**User Impact:**
When non-text content is used in a consistent way to identify components with the same functionality, people with difficulty reading text or detecting text alternatives can interact with the Web without depending on text alternatives.
User may assume that both controls have the same functionality.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={2a3e26b6-fe3e-4205-8370-ee24fa1764ac}

**Attachment:**
![bug 129_consistent identification issue](https://user-images.githubusercontent.com/38521133/44713667-3d695480-aad1-11e8-87d3-310e8f545395.png)

**Comment by jhkimnew** [comment]
@shhsu, FYI, I am going to close this issue because you are going to update the Files feature with the new style and the "More" button won't be used.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
